src/main/java/bio/terra/service/snapshotbuilder/query/TablePointer.java

```diff
@@ -2,21 +2,14 @@
 
 import java.util.List;
 
-public record TablePointer(String tableName, Filter filter, String sql) implements SqlExpression {
+public record TablePointer(String tableName, Filter filter) implements SqlExpression {
 
   public static TablePointer fromTableName(String tableName) {
-    return new TablePointer(tableName, null, null);
-  }
-
-  public static TablePointer fromRawSql(String sql) {
-    return new TablePointer(null, null, sql);
+    return new TablePointer(tableName, null);
   }
 
   @Override
   public String renderSQL(SqlRenderContext context) {
-    if (sql != null) {
```

Comment:
This case was only used in test code, so I rewrote the tests so it wasn't needed anymore.